ebos/ecltransmissibility_impl.hh

```diff
@@ -483,6 +507,42 @@ update(bool global, const std::function<unsigned int(unsigned int)>& map, const
 
                 diffusivity_[isId(elemIdx, outsideElemIdx)] = diffusivity;
            }
+
+           // update the "dispersivity half transmissibility" for the intersection
+            if (updateDispersivity) {
+
+                Scalar halfDispersivity1;
+                Scalar halfDispersivity2;
+
+                computeHalfDiffusivity_(halfDispersivity1,
+                                        faceAreaNormal,
+                                        distanceVector_(faceCenterInside,
+                                                        intersection.indexInInside(),
+                                                        elemIdx,
+                                                        axisCentroids),
+                                        dispersion_[elemIdx]);
+                computeHalfDiffusivity_(halfDispersivity2,
+                                        faceAreaNormal,
+                                        distanceVector_(faceCenterOutside,
+                                                        intersection.indexInOutside(),
+                                                        outsideElemIdx,
+                                                        axisCentroids),
+                                        dispersion_[outsideElemIdx]);
+
+                applyNtg_(halfDispersivity1, insideFaceIdx, elemIdx, ntg);
+                applyNtg_(halfDispersivity2, outsideFaceIdx, outsideElemIdx, ntg);
+
+                //TODO Add support for multipliers
+                Scalar dispersivity;
+                if (std::abs(halfDispersivity1) < 1e-30 || std::abs(halfDispersivity2) < 1e-30)
+                    // avoid division by zero
```

Comment:
Probably ok, but I think this eps should be moved to a common place. Also in similar code for thermalconductivity I think <= 0 is used.

Reply:
This is just copy from diffusion, I agree this gathering of all epsilons should be in a common place, but maybe this should be done in a separate PR for all parameters.